### ISSUE TITLE
Migrate commans Cleo command-line 

### ISSUE THREAD

**Issue by estyxx** [autopub/autopub#11]
Refactoring/improve the commands using [Cleo](https://cleo.readthedocs.io/en/latest/index.html) like poetry does [see commands](https://github.com/python-poetry/poetry/tree/master/poetry/console/commands).

I think Cleo is a nice tool to write command line commands like poetry, and it enables to write tests! I think we could think to migrate to it in order to clean up some scripts and start to write some tests :)

What do you think @justinmayer?

**Comment by marcoacierno** [comment]
I like the idea.

I wonder if we can write some tests now so that when we start refactoring for Cleo we have some sanity checks, what do you think?

**Comment by estyxx** [comment]
Yeah, I can start prototype some refactoring and see if can be an improvement :) 
Otherwise, until now I always test locally...
